frameworks/Haskell/servant/hasql/src/ServantBench.hs

```diff
@@ -148,6 +164,32 @@ multipleDb pool gen mQueryId = do
 
 -- * Test 4: Fortunes
 
+type (>) a b = a Html.> b
+type (#) a b = a Html.# b
+type FortunesHtml
+  = (('Html.DOCTYPE Html.> ())
```

Comment:
This is the first time I've seen the *structure of the HTML* in a servant API type! :)